xml/operations-configuring-identity-identity_limitations.xml

```diff
@@ -322,8 +322,17 @@ for production use until a future OpenStack release.</li> -->
    </listitem>
    <listitem>
     <para>
-     Only WebSSO federation using the SAML method is supported in
-     &product; . OpenID-based federation is not currently supported.
+     The &productname; WebSSO function with OpenID method was tested with
+     Google OAuth 2.0 APIs, which conforms to the OpenID Connect specification.
+     The interaction between Keystone and external Identity Provider (IDP) is
+     handled by the Apache2 <link xlink:href="https://github.com/zmartzone/mod_auth_openidc">auth_mod_openidc</link> module. Please consult with the specific OpenID Connect vendor
+     on whether they support <link xlink:href="https://github.com/zmartzone/mod_auth_openidc">auth_mod_openidc</link>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Both SAML and OpenID methods are supported for WebSSO federation in
+     &product; .
```

Comment:
We use " &productname;" above when talking about the support in the product, so to be consistent we should probably use " &productname;" here too.

Reply:
I am confused about the use of "&productname;" versus "&product;" in the doc too. For Cloud 9, "&productname;" is mapped to "SUSE OpenStack Cloud", whereas "&product;" is mapped to "SUSE OpenStack Cloud 9". I see them used interchangeably throughout the doc.

Reply:
We probably need guidance from Carl for this. @csymons-suse how should this be handled? My suggestion is to rarely use "SUSE OpenStack Cloud 9" so that we don't have to change those all to "Cloud 10" for the next version - it just saves work.

Reply:
Carl's out until next week - @bryanstephenson & @guangyee using &product; is fine. We don't have to change everything to Cloud 10 for the next version, as everything will be completely different regardless, plus, we have an entity file which we update and that automatically applies the changes: https://github.com/SUSE-Cloud/doc-cloud/blob/master/xml/entity-decl.ent#L29 So that is not a concern.

Contextually, either is fine. But as this is support, we should indicate what specific version to support - so the use of "&product;" is preferable.